### ISSUE TITLE
Customize each row with different menu items, is it possible ?  

### ISSUE THREAD

**Issue by sumitsahoo** [baoyongzhang/SwipeMenuListView#52]
Lets say I want 3 menu items for one row but 2 for another or a different image for other row. Is it possible ?

**Comment by baoyongzhang** [comment]
Yes. https://github.com/baoyongzhang/SwipeMenuListView#user-content-create-different-menu

**Comment by Gurvinder963** [comment]
not working.

menu.getViewType(); is always returing 0 even put  get itemviewtype in baseAdapter

@Override
    public int getItemViewType(int position) {

```
    return position%2;
    }
```
